jobbergate-cli/jobbergate_cli/config.py

```diff
@@ -57,6 +57,7 @@ class Settings(BaseSettings):
     OIDC_DOMAIN: str
     OIDC_AUDIENCE: str
     OIDC_CLIENT_ID: str
+    OIDC_USE_HTTPS: bool = True
```

Comment:
What would be the scenarios for https and http?

Reply:
HTTPS should be used whenever you are hosting your APIs publicly. However, when you are using internal webservers (like local debug stacks), then it's nice to use HTTP so you don't have to worry about certs and all that mess.

So, you default to HTTPS, but allow your user to force HTTP if they ask for it.